### PR TITLE
Update configure-s-mime-settings-for-outlook-web-app.md

### PR DESCRIPTION
**Pull request by msbemba**
Added  AAD joined devices to the considerations

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2411

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/security/office-365-security/configure-s-mime-settings-for-outlook-web-app.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ For detailed syntax and parameter information, see [Get-SmimeConfig](https://doc
...
 
-To use S/MIME in Outlook on the web in the new [Microsoft Edge](https://www.microsoft.com/windows/microsoft-edge) web browser, you (or another admin) must set and configure the Microsoft Edge browser policy named **ExtensionInstallForcelist** to install the Microsoft S/MIME extension in new Microsoft Edge. The policy value is `maafgiompdekodanheihhgilkjchcakm;https://outlook.office.com/owa/SmimeCrxUpdate.ashx`. And note that applying this policy requires domain-joined computers, so using S/MIME in the new Microsoft Edge browser effectively requires domain-joined computers.
+To use S/MIME in Outlook on the web in the new [Microsoft Edge](https://www.microsoft.com/windows/microsoft-edge) web browser, you (or another admin) must set and configure the Microsoft Edge browser policy named **ExtensionInstallForcelist** to install the Microsoft S/MIME extension in new Microsoft Edge. The policy value is `maafgiompdekodanheihhgilkjchcakm;https://outlook.office.com/owa/SmimeCrxUpdate.ashx`. And note that applying this policy requires domain-joined OR Azure AD Joined Devices, so using S/MIME in the new Microsoft Edge browser effectively requires domain-joined or Azure AD-joined devices.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-s-mime-settings-for-outlook-web-app.md]:
```suggestion
To use S/MIME in Outlook on the web in the new [Microsoft Edge](https://www.microsoft.com/windows/microsoft-edge) web browser, you (or another admin) must set and configure the Microsoft Edge browser policy named **ExtensionInstallForcelist** to install the Microsoft S/MIME extension in the new Microsoft Edge. The policy value is `maafgiompdekodanheihhgilkjchcakm;https://outlook.office.com/owa/SmimeCrxUpdate.ashx`. And note that applying this policy requires domain-joined or Azure AD-joined devices, so using S/MIME in the new Microsoft Edge browser effectively requires domain-joined or Azure AD-joined devices.
```

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/configure-s-mime-settings-for-outlook-web-app.md]: Please change "in new Microsoft Edge" to "in the new Microsoft Edge", and change capitalization in the last sentence as indicated.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks

**Pull request merged**